udata/core/spatial/translations.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+# Here to force translations by gettext
+_ = lambda s: s
+
+TRANSLATIONS = (
```

Comment:
There was two alternative: the easy one in which we manually maintain a label list in uData.
The other one is to import translations from GeoZones.
I add to manually test the translations (on card) to check that every string is translated.